### PR TITLE
Add option to change the retain policy of the storage class

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [ ] Link this PR to related issues.

### CODE REVIEW

File: class/defaults.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ parameters:
         class: ssd
         fstype: ext4
         volumeexpansion: true
+        retainpolicy: Retain
```
Comment (megian) [inline class/defaults.yml]: Would keep the default Delete in the component. We might change that for certain customers.

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 2]
```diff
@@ -123,6 +123,13 @@ type:: boolean
 
 Should volume expansion be allowed with this storage class.
 
+=== `retainpolicy`
+
+[horizontal]
+type:: string
+
```
Comment (megian) [inline docs/modules/ROOT/pages/references/parameters.adoc:130]: default should be added here

**Review by megian — CHANGES_REQUESTED** [review]

File: class/defaults.yml (hunk 1 above)
Comment (DebakelOrakel) [reply class/defaults.yml]: ah shoot, testing artefact

File: docs/modules/ROOT/pages/references/parameters.adoc (hunk 2 above)
Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/references/parameters.adoc:130]: It is an optional parameter, will remove it from the defaults.

**Review by megian — APPROVED** [review]

**Pull request merged**